treeherder/config/settings.py

```diff
@@ -20,17 +20,17 @@
 
 env = environ.Env()
 
-#Checking for OS type
+# Checking for OS type
 IS_WINDOWS = "windows" in platform.system().lower()
 
 # Top Level configuration
 DEBUG = env.bool("TREEHERDER_DEBUG", default=False)
 
 #  Default environment variables check
-REQUIRED_PRODUCTION_ENV_VARIABLES = ["TREEHERDER_DJANGO_SECRET_KEY","BROKER_URL","DATABASE_URL","REDIS_URL","TREEHERDER_DEBUG","NEW_RELIC_DEVELOPER_MODE"]
+REQUIRED_PRODUCTION_ENV_VARIABLES = ["TREEHERDER_DJANGO_SECRET_KEY", "BROKER_URL", "DATABASE_URL", "REDIS_URL", "TREEHERDER_DEBUG", "NEW_RELIC_DEVELOPER_MODE"]
```

Comment:
Could you please break this over multiple lines? It does not have to be one per line but it is quite long as it is.